test/smoke/src/areas/positron/output/consoleOutputLog.test.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { Application, Logger, PositronPythonFixtures, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+
+export function setup(logger: Logger) {
+	describe('Output', () => {
+		// Shared before/after handling
+		installAllHandlers(logger);
+
+		describe('Console Output Log - Python', () => {
+			before(async function () {
+				const app = this.app as Application;
+				const pythonFixtures = new PositronPythonFixtures(app);
+				await pythonFixtures.startPythonInterpreter();
```

Comment:
I wonder if this is a common enough pattern we should create a utility function for it.

Reply:
E.g. 
```ts

export class PositronRFixtures {

	static async SetupFixtures(app: Application) {
		const fixtures = new PositronRFixtures(app);
		await fixtures.startRInterpreter();
	}
	...
}
```

And then 
```ts
...
before(async function () {
    await PositronRFixtures.SetupFixtures(this.app as Application);
});
...
```
Probably premature optimization though.

Reply:
Yeah, those fixtures are used in most tests.  i will change to use the static approach.